spec/features/interview_spec.rb

```diff
@@ -1,6 +1,11 @@
 require "rails_helper"
+require 'selenium-webdriver'
 
 RSpec.describe "Chat", type: :feature do
+  before '@javascript' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.